src/rules/custom-property-no-empty-line-before/README.md

```diff
@@ -0,0 +1,83 @@
+# custom-property-no-empty-lines-before
+
+Disallow empty lines before the custom properties
+
+```css
+    a { 
+        --foo-bar: 1px;
+                            /* ← */
+        --foo-baz: 2px;     /* ↑ */   
+    }                       /* ↑ */
+     /**                       ↑
+     *                 This line */
+```
+
+## Options
+
+### `true`
+
+The following patterns are considered warnings:
+
+```css
+a { 
```

Comment:
The blocks in each of these examples should be indented by two spaces. They seem to be tabs.

If you're not already, trying using an [editorconfig](http://editorconfig.org/#download) plugin for your editor as this will fix the problem for you.